pkg/metrics/metrics.go

```diff
@@ -133,6 +137,16 @@ var (
 		Name:      "upgrade_window_breached",
 		Help:      "Failed to commence upgrade during the upgrade window",
 	}, []string{nameLabel})
+	metricUpgradeControlPlaneTimeout = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Subsystem: metricsTag,
+		Name:      "controlplane_timeout",
```

Comment:
This name doesn't sounds good to me, we probably should revisit the existing metrics names to have a better pattern like "managed_upgrade_xxx"

Reply:
the metricsTag is the prefix for all the metrics.
So it will be `upgradeoperator_xxx` now. If the `upgradeoperator` does not make sense.
We can change it in const

Reply:
thanks, missed that